test/services/ApprovalServiceSpec.scala

```diff
@@ -37,7 +37,7 @@ class ApprovalServiceSpec extends UnitSpec with MockFactory {
 
```

Comment:
I think this class is a little on the light side. As the main test process is actually empty the page builder can't be doing anything. Perhaps an update for a later day.

Reply:
yes